lbrynet/core/Wallet.py

```diff
@@ -98,7 +102,8 @@ def get_claim_metadata_for_sd_hash(self, sd_hash):
             return defer.succeed(None)
 
     def update_claimid(self, claim_id, name, claim_outpoint):
-        self.claimids[(name, claim_outpoint['txid'], claim_outpoint['nout'])] = claim_id
+        self.claimids[
+            (name, claim_outpoint['txid'], claim_outpoint['nout'])] = claim_id
```

Comment:
Can you remove this kind of whitespace? We typically only split up lines like this if they're longer than 100 characters or if it helps readability.